src/common/components/grid/cell-renderer/entity-type.cell-renderer.tsx

```diff
@@ -28,6 +28,10 @@ export const EntityTypeRenderer: React.FC<IEntityTypeCellRendererParams> = (prop
       icon = 'mc-icon-Bests';
       value = 'BEST';
       break;
+    case 'LayerDEMRecord':
```

Comment:
@assafchamoy did something like that but on ProductType we should normalize it!!!